paper/paper.md

```diff
@@ -0,0 +1,105 @@
+
+In this paper, we present the MapReader release at the conclusion of the Living with Machines project, which supported the development of the software and associated research. This release represents the culmination of extensive work to improve MapReader's usability, especially through clear documentation and tutorials.
+
+![MapReader modules and input-outputs. Credit: Rosie Wood.\label{fig:modules}](https://hackmd.io/_uploads/HJWJatQEa.png)
+
+
+# Statement of need
+
+Since the 1990s, map libraries have been scanning maps and creating digital collections of these images. In 2023, there are more than a million images of maps in digital libraries and archives around the world, and yet it is very difficult for anyone to do more than browse them in a web viewer.
+
+MapReader makes it possible to ask questions of thousands of digitized maps at a time, a fundamentally different intellectual experience from both the traditional manner of viewing a few maps at a time on a reading room table and the act of visually scanning digital files sequentially. Inspired by the possibility of seeing series maps stitched together in seamless layers, such as the National Library of Scotland Ordnance Survey map viewing interface, MapReader takes the next step by transforming the experience of working with maps from surface exploration to critical investigation `[Hosseini_maps]`.
+
+
+# Related Work
+
+MapReader is among the first end-to-end pipeline for processing historical maps and other images that was designed to lower barriers to experimenting with computer vision in answering research questions about large image datasets. Other projects are emerging which are performing similar research tasks with the visual content in historical map collections `[@Petitpierre; @Combes]`, and of course other tools, like the Distant Viewing Toolkit `[@Arnold]`, address similar needs for other kinds of media. As part of a collaboration between Machines Reading Maps and the David Rumsey Historical Map Collection, the Knowledge Computing Lab released mapKurator `[@mapkurator]`, which takes map image input and creates geojson files of all text on each map.
+
+
+# Documentation
+
+MapReader aims to build computational skills among historians. Our extensive work on documentation and training, including substantial updates to MapReader since `@Hosseini_mapreader`, reflect this commitment. As historians explore the possibilities of computational methods for novel historical research, MapReader models how computational tools can unlock difficult to use primary sources and how we can embrace open research practices as a way to encourage learning. We welcome contributions and requests for new documentation or tutorials.
+
+Our documentation includes:
+- About MapReader: A basic introduction to the software and its origins
+- Events: Activities where the community can engage with MapReader
+- Project CV: Papers, talks, workshops, etc. delivered by the MapReader team
```

Comment:
CV / Computer Vision / Curriculum Vitae ... maybe we can choose another title here?

Reply:
Good point.